RELEASE_NOTES.md

```diff
@@ -1,8 +1,11 @@
-# zcbor v. 0.5.99
+# zcbor v. 0.5.1
 
 Any new bugs, requests, or missing features should be reported as [Github issues](https://github.com/NordicSemiconductor/zcbor/issues).
 
 ## Improvements:
+ * zcbor_encode/zcbor_decode: Add new functions zcbor_int_encode() and zcbor_int_decode() which accept the integer size as an argument.
+ * zcbor.py: Use zcbor_int_encode() and zcbor_int_decode() when operating directly on _choice enums, since enums can be any size, depending on compiler and project. This removes the restrictions that enums must be 32 bits.
+ * requirements.txt: Change PyYAML requirement from 6.0.0 to 5.4.1 to be more compatible.
```

Comment:
be more compatible with what?

Reply:
So it can be installed together with packages that require PyYAML<6.0.0.

This was done so it can be used in the sdk-nrf CI which has fixed versions for everything, and has PyYAML at 5.4.1.

Reply:
BTW this comes from #236

Reply:
I changed the wording slightly in #236